de-lib/src/main/java/org/iplantc/de/theme/base/client/tools/ToolMessages.java

```diff
@@ -86,6 +86,8 @@
     String manageSharing();
 
     String done();
+
+    String entryPoint();
```

Comment:
I agree with @tedgin that we should add java docs to the interface methods (at least just for the methods added in this PR). It could also just be a copy of the [endpoint docs](https://github.com/cyverse-de/apps/blob/2.12.0/src/apps/routes/schemas/containers.clj#L56).